client/src/components/Header/Header.tsx

```diff
@@ -0,0 +1,21 @@
+import React, { FC } from 'react'
+import icon from '../../images/icon.svg'
+
+import './Header.scss'
+
+export interface HeaderProps {
+  title?: string
+  subtitle?: string
+}
+
+const Header: FC<HeaderProps> = ({ title, subtitle }) => (
+  <header className='Header'>
+    <img className='Header--Icon' src={icon} alt='Logo de Poule poule' />
```

Comment:
[This article](https://a11yproject.com/posts/alt-text/) suggests that `alt` attributes should not start with "picture of ".
```suggestion
    <img className='Header--Icon' src={icon} alt='Poule poule' />
```

I would even say this is a decorative image and should have an empty `alt` but that's just me...
```suggestion
    <img className='Header--Icon' src={icon} alt='' />
```

Reply:
Yeah, as it is "supposed" to be a logo, I added it. "Picture of a logo de poule poule" makes sense to me. But it's maybe better without.

Reply:
Oh yeah you totally right: it's decorative. And it would double the "Poule poule" with it (image alt + title).